pubspec.yaml

```diff
@@ -5,8 +5,8 @@ homepage: https://github.com/ubuntu/yaru_widgets.dart
 version: 1.1.3
 
 environment:
-  sdk: ">=2.17.0 <3.3.2"
-  flutter: ">=3.3.2"
+  sdk: ">=2.17.0 <3.0.0"
```

Comment:
this is the Dart SDK i.e. the language version. anything less than 3.0 is accepted but 3.0 is not because it's going to incompatible